### PR TITLE
DBAAS-599

### PR DESCRIPTION
**Pull request by olavtar**
Fixed the message for the tooltip displayed in case of the Connection failure.

Signed-off-by: Olga Lavtar <olavtar@redhat.com>

### CODE REVIEW

**Review by jeremyary — APPROVED** [review]
lgtm, thanks!

File: src/components/adminDashboard.jsx [hunk 1]
```diff
@@ -225,7 +225,11 @@ const AdminDashboard = () => {
           inventory.status?.instances?.map((instance) => (instance.provider = inventory.spec?.providerRef?.name))
           obj.instances = inventory.status?.instances
           if (specSyncedCondition.status === 'False' || inventoryReadyCondition.status === 'False') {
-            obj.alert = 'alert'
+            if (specSyncedCondition.reason === 'AuthenticationError') {
+              obj.alert = 'Can not establish a connection to this database instance.'
+            } else {
```
Comment (redhatHameed) [inline src/components/adminDashboard.jsx:230]: for other reason we are keeping the generic message right ?

Comment (olavtar) [reply src/components/adminDashboard.jsx:230]: @redhatHameed yes.

**Review by redhatHameed — COMMENTED** [review]
LGTM

**Review by yzhao583 — APPROVED** [review]
LGTM thanks

**Pull request merged**